azure/iac/sql_servers.rego

```diff
@@ -53,3 +53,53 @@ sql_public_access_disabled_metadata := {
     "Policy Help URL": "",
     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/servers"
 }
+
+
+
+
+
+
+# https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/2014-04-01/servers/administrators
+
+#
+
+# PR-AZR-0133-ARM
+
+default sql_server_login = null
+
+
+azure_issue["sql_server_login"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.sql/servers/administrators"
+    lower(resource.properties.login) != "admin"
+    lower(resource.properties.login) != "administrator"
```

Comment:
checking property existence would be a good idea whether its required or not.

Reply:
if customer somehow forgot to add the objects in arm template which they required, he/she can't deploy that
If it was not so we should check existence all the objects that they are required

Reply:
@shahinshirvani Yes they can't deploy and we are doing the validation before deployment. is n't it better if we can notify customer before deployment, that a required property is missing from your template?
I think the sole purpose of these policies are to give customer a hint/suggestion/fix about incorrect template configuration.

Reply:
@rezoan @shahinshirvani we cannot check the existence of all the required values from the ARM template. but we can do that for the attributes we are constructing our policies based on them

Reply:
fixed